index.html

```diff
@@ -82,14 +95,15 @@ <h4>Tonic</h4>
                     <button class="btn" type="submit">See Project</button>
                 </div>
             </div>
-            <div class="container">
+            <div class="container1">
                 <div class="snapshoot1">
-                    <img class="project_photo" src="images/pic1.png" alt="Project's image" />
+                    <img class="snapshootimg" src="images/desktop-images/img2.png" alt="project image">
+                   <img class="project_photo" src="images/pic1.png" alt="Project's image" /> 
                 </div>
                 <div class="projects projects1">
```

Comment:
- [ ] Some projects are having too much margin on the left, kindly fix that, and replicate the Figma design as much as possible